### ISSUE TITLE
Remove nix in favour of venv

### ISSUE THREAD

**Issue by stickyPiston** [svsticky/undead-mongoose#61]
Currently, we use nix to manage python packages, but a more standard approach supported better by IDEs is to use python virtual environments. In my opinion, nix adds quite a bit of complexity (and flexibility) that we don't really need in the mongoose. Therefore I'd like to propose to use a less flexible but more simple tool for managing python packages: [virtual environments](https://docs.python.org/3/library/venv.html).

**Comment by SilasPeters** [comment]
I agree. We should use venv and potentially virtualenv. Nix is great, but too complicated for this

**Comment by SamStaijen** [comment]
I think nix is great, but the learning curve looks too much like ln(x) in my opinion. I would prefer something like venv as well. 

I also volunteer as a tester, if you are able to find someone to implement it that is :).

tl;dr: I Agree!

**Comment by stickyPiston** [comment]
closed in #64
